pkgs/applications/editors/android-studio/common.nix

```diff
@@ -198,8 +198,11 @@ in runCommand
       # binaries are also distributed as proprietary software (unlike the
       # source-code itself).
       platforms = [ "x86_64-linux" ];
-      maintainers = with maintainers; ([ ]
-       ++ optional (channel == "stable") primeos);
+      maintainers = with maintainers; {
+        stable = [ primeos ];
+        beta = [ galagora ];
+        canary = [ ];
+      }."${channel}";
```

Comment:
There's also a `dev` channel which is why the CI check fails (annoyingly `dev` and `canary` are identical so I guess we could e.g. also drop `canary`).

Reply:
Update: Just realized that at least https://developer.android.com/studio/preview/ uses Canary so we should probably drop `dev` instead of `canary` if we choose that solution.

Reply:
Fixed.